src/AcceptanceTests/Infrastructure/DefaultServer.cs

```diff
@@ -38,6 +38,7 @@ public Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDescriptor,
             var recoverability = configuration.Recoverability();
             recoverability.Delayed(delayed => delayed.NumberOfRetries(0));
             recoverability.Immediate(immediate => immediate.NumberOfRetries(0));
+            configuration.SendFailedMessagesTo("error");
 
             var transportConfig = configuration.UseTransport<MsmqTransport>();
```

Comment:
I did it here. Switching other downstreams is probably a bigger tasks and deserves its own issue, that's all I was trying to say.